client/src/app/shared/buttons/action-dropdown.component.ts

```diff
@@ -4,9 +4,12 @@ import { GlobalIconName } from '@app/shared/images/global-icon.component'
 export type DropdownAction<T> = {
   label?: string
   iconName?: GlobalIconName
+  description?: string
+  title?: string
   handler?: (a: T) => any
   linkBuilder?: (a: T) => (string | number)[]
   isDisplayed?: (a: T) => boolean
+  actions?: DropdownAction<T>[]
```

Comment:
where is it used?

Reply:
it is not, I shouldn't have included it: that was just an artifact of me trying to have nested dropdowns.

Reply:
it is not used. It should have been removed.